src/CharacterStates/CharacterState.cpp

```diff
@@ -6,6 +6,10 @@ CharacterState::CharacterState(int currentFrame, int framesAmount) {
     this -> framesAmount = framesAmount;
 }
 
+CharacterState::CharacterState() {
+
```

Comment:
Why is this necessary? (It is not necessary to be changed, but I do not know what purpose this accomplish)